replicator/src/main/scala/com/evolutiongaming/kafka/journal/replicator/TopicReplicator.scala

```diff
@@ -92,9 +92,8 @@ object TopicReplicator {
     val topicFlowOf: TopicFlowOf[F] = {
       (topic: Topic) => {
         for {
-          journal  <- journal.journal(topic)
-          pointers <- journal.pointers.toResource
-          cache    <- cacheOf[Partition, PartitionFlow](topic)
+          journal <- journal.journal(topic)
```

Comment:
most of changes in this file are about formatting :(

Reply:
I've fixed it but we have to introduce common formatter defined in project sources (not IntelliJ config), otherwise doing manual formatting each time is annoying